src/modules/mail/templates/email.templates.ts

```diff
@@ -13,3 +13,25 @@ export const sendCodeMessage = (userName: string, code: string) => {
           </div>
         `;
 };
+
+export const sendWelcomeMessage = (
+  userName: string,
+  appUrl: string // Recibe la URL base como argumento
+) => `
+  <div style="font-family: Arial, sans-serif; line-height: 1.6; color: #333; text-align: center; max-width: 600px; margin: auto; padding: 20px;">
+    <h1 style="color: #007bff;">Welcome to M2!</h1>
+    <p>Hello ${userName},</p>
+    <p>You’re all set! To secure your account, you can change your password using the button below:</p>
+    <div style="margin: 20px 0;">
+      <a href="${appUrl}/reset-password" 
+         style="display: inline-block; background-color: #007bff; color: white; text-decoration: none; padding: 10px 20px; border-radius: 5px; font-size: 16px; font-weight: bold;">
+        Reset Password
+      </a>
+    </div>
+    <p>If you have any questions, feel free to contact us anytime.</p>
+    <p>Thank you,</p>
+    <p>The M2 Team</p>
+  </div>
```

Comment:
also there is not anymore M2 team, please update it to OSM team

Reply:
OK